### PR TITLE
chore: update some information about CPU StressChaos

### PR DESCRIPTION
**Pull request by FingerLeader**
Signed-off-by: FingerLeader <wanxfinger@gmail.com>

### CODE REVIEW

**Comment by en-jin19** [comment]
@FingerLeader Could you please invite a tech reviewer for this PR?

Besides, please update the same English docs of v2.1.2 and v2.0.6 and Chinese docs of the current version,  v2.1.2, and v2.0.6? thanks!

**Comment by FingerLeader** [comment]
> @FingerLeader Could you please invite a tech reviewer for this PR?
> 
> Besides, please update the same English docs of v2.1.2 and v2.0.6 and Chinese docs of the current version, v2.1.2, and v2.0.6? thanks!

Thank you for reminding, I have updated other docs, and PTAL /cc @STRRL

**Review by en-jin19 — APPROVED** [review]
LGTM

File: docs/simulate-heavy-stress-on-kubernetes.md [hunk 1]
```diff
@@ -78,12 +78,12 @@ The fields in the YAML configuration file are described in the following table:
 
 | Parameter | Type   | Description                                              | Default value | Required | Example |
 | --------- | ------ | -------------------------------------------------------- | ------------- | -------- | ------- |
-| workers   | int    | Specifies the number of threads that apply memory stress |               | Yes      | `1`     |
+| workers   | int    | Specifies the number of threads that apply CPU stress |               | Yes      | `1`     |
```
Comment (STRRL) [inline docs/simulate-heavy-stress-on-kubernetes.md:81]: I do not think this description for `MemoryStressor.workers` is correct. :thinking:

File: docs/simulate-heavy-stress-on-kubernetes.md [hunk 2]
```diff
@@ -78,12 +78,12 @@ The fields in the YAML configuration file are described in the following table:
...
 | size      | string | Specifies the maximum memory size available for total    |               | No       | `256MB` |
```
Comment (STRRL) [inline docs/simulate-heavy-stress-on-kubernetes.md:82]: Please explain that `size` is the expected allocated memory for **each worker**, and the final sum of allocated memory size is `workers * size`.

And please also introduce that we could use percentage `%` in here.

**Review by STRRL — CHANGES_REQUESTED** [review]
It seems https://github.com/chaos-mesh/chaos-mesh/pull/2623 has not been cherry-picked into release-2.0 and release-2.1, so changes on versioned documentation are NOT required.

And it's better to introduce why we use `memStress` instead of `stress-ng`, what's the different behavior between them.

File: docs/simulate-heavy-stress-on-kubernetes.md (hunk 1 above)
Comment (FingerLeader) [reply docs/simulate-heavy-stress-on-kubernetes.md:81]: This PR is not for [chaos-mesh/chaos-mesh#2623](https://github.com/chaos-mesh/chaos-mesh/pull/2623), just fix some typo for the origin implementation of StressChaos. I will add details for #2623 in another PR.

**Comment by FingerLeader** [comment]
I have update docs of stressChaos in another PR, both for new changes and old typo. This PR can be closed.

**Pull request closed without merging**